### DISCUSSION TITLE
How do I make custom arrow types?

### DISCUSSION THREAD

**Discussion by Ronezkj15** [KadeArchive/Kade-Engine#809] [Q&A]
I've been trying to do this for about a couple hours now and the best I can do is a reskin that can't do anything unique.

How do I make arrow types like the fire and angel notes from Tricky, with different skins and functionality?

Any help would greatly be appreciated!

**Comment by puyoxyz** [comment]
you could look at the [Tricky source code](https://github.com/kadedev/trickster) and see how they did it?
